test/README.md

```diff
@@ -0,0 +1,14 @@
+# *WIP*
+
+This is the place to put your unit tests.
+
+# Unit tess
+
+# E2E (Cypress)
+[Cypress Readme](/cypress/README.md)
+
+## Difference of unit testing and E2E
+- Unit: Test one part of a component/code in isolation (--> "a unit")
+  - We are using Jest (alternatives: Jasmine, Mocha)
+- E2E: Test many components/code interacting (--> End to End)
+  - We are using Cypress (alternatives: Playwright, others)
```

Comment:
[In this comment](https://github.com/PrimeDAO/prime-deals-dapp/pull/63#discussion_r780703686) you mentioned
> Put this in the context of other tests that don't require a UI?

Can you rephrase? Is this explanation here enough?